waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -134,14 +153,22 @@
        ;; for backward compatibility where the revision timestamp is missing we cannot use the destructuring above
        rs-annotations (get-in replicaset-json [:metadata :annotations] nil)
        rs-pod-annotations (get-in replicaset-json [:spec :template :metadata :annotations] nil)
-       rs-containers (vec (map :name (get-in replicaset-json [:spec :template :spec :containers])))
+       containers (get-in replicaset-json [:spec :template :spec :containers])
+       rs-containers (vec (map :name containers))
```

Comment:
Nitpick: I see the old line was already written this way, but since you're touching the line anyway...
`(vec (map ...))` → `(mapv ...)`

Reply:
Done.